### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Abidoyinsola1**
At HTML changes:
 - Added header, main and footer tag sections of the page.

At CSS changes: 
- Styled the header tag, including the nav.
- Added background image
- Styled main section
- Styled  the footer section

These were done to be as the given webpage for the project.

### CODE REVIEW

**Comment by tirthajyoti-ghosh** [comment]
# Pull Request under review by T. Ghosh (TSE)

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🚨

## Hello @Abidoyinsola1 and @somoye123 🖐
## Awesome project team ✨👌

### However, we still need to work on some changes before we can move ahead.

- The `nav` section should have 9 elements including the input field. Currently, you have only 8. Also, add a bit more space above it and decrease its size to match the original.
  
   Right now 👇

   Annotation 2020-06-11 231227

   Original 👇
  
   Annotation 2020-06-11 231253

-  Decrease the size and font-weight of `header-text` to match the original.

*I have also left some reviews below. Be sure to check them.*

File: index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      <nav>
+        <div class="nav-items">
+          <a href=""><i class="fa fa-apple"></i></a>
+        </div>
+        <div class="nav-items"><a href="">Store</a></div>
+        <div class="nav-items"><a href="">Mac</a></div>
+        <div class="nav-items"><a href="">iPad</a></div>
+        <div class="nav-items"><a href="">iPhone</a></div>
+        <div class="nav-items"><a href="">iTunes</a></div>
+        <div class="nav-items"><a href="">Support</a></div>
```
Comment (tirthajyoti-ghosh) [inline index.html]: - The `href` attribute must not be empty. Add `#` -> `href="#"`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+          <a href="#!"> reseller</a>.
+        </p>
+        <ul>
+          <li><a href=""> Site Map</a></li>
+          <li><a href="">Hot News</a></li>
+          <li><a href="">RSS Feeds</a></li>
+          <li><a href="">Media Info</a></li>
+          <li><a href="">Environment</a></li>
+          <li><a href="">Job Opportunities</a></li>
+          <li><a href="">Contact Us</a></li>
```
Comment (tirthajyoti-ghosh) [inline index.html]: - The `href` attribute must not be empty. Add `#` -> `href="#"`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+          >Watch the "Your Verse" film and explore the stories
+          <i class="fa fa-play-circle"></i
+        ></a>
+      </section>
+    </header>
+    <main>
+      <a href="#!"><img src="asset/image/apple-1.jpg" alt="" /></a>
+      <a href="#!"><img src="asset/image/apple-2.jpg" alt="" /></a>
+      <a href="#!"><img src="asset/image/apple-3.jpg" alt="" /></a>
+      <a href="#!"><img src="asset/image/apple-4.jpg" alt="" /></a>
```
Comment (tirthajyoti-ghosh) [inline index.html]: - Remove `!` in `href`. It should be -> `href="#"`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    </main>
+    <footer>
+      <div class="footer">
+        <p>
+          Shop the <a href="#!"> Apple Online Store</a> (1-800-MY-APPLE), visit
+          an <a href=""> Apple Retail Store</a>, or find a
+          <a href="#!"> reseller</a>.
```
Comment (tirthajyoti-ghosh) [inline index.html]: - Remove `!` in `href`. It should be -> `href="#"`

File: index.html [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+      <a href="#!"><img src="asset/image/apple-1.jpg" alt="" /></a>
+      <a href="#!"><img src="asset/image/apple-2.jpg" alt="" /></a>
+      <a href="#!"><img src="asset/image/apple-3.jpg" alt="" /></a>
+      <a href="#!"><img src="asset/image/apple-4.jpg" alt="" /></a>
+    </main>
+    <footer>
+      <div class="footer">
+        <p>
+          Shop the <a href="#!"> Apple Online Store</a> (1-800-MY-APPLE), visit
+          an <a href=""> Apple Retail Store</a>, or find a
```
Comment (tirthajyoti-ghosh) [inline index.html]: - The `href` attribute must not be empty. Add `#` -> `href="#"`

**Review by tirthajyoti-ghosh — COMMENTED** [review]
Apply these changes and request another code review.

Happy coding 👨‍💻

**Comment by BrittanyBlake** [comment]
### Pull Request under review by Brittany Blake (TSE)

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi Team! 👋 
You have done a great job implementing the changes requested by the previous TSE! 👍 You have also done a wonderful job with your project so far! It looks really good! 👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- [ ] The search bar in your navigation is extending outside of the navigation bar. Please compare your clone with the original page and fix this.
Your clone 👇 
Screen Shot 2020-06-11 at 9 33 04 PM
The original page 👇 
Screen Shot 2020-06-11 at 9 39 41 PM

- [ ] As was requested by the previous TSE, please decrease your font-size and weight for `header-text`.
The original page 👇 
Screen Shot 2020-06-11 at 9 41 52 PM
Your clone 👇 
Screen Shot 2020-06-11 at 9 41 57 PM




****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

**Review by andresporras3423 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
**Hello @Abidoyinsola1 @somoye123** 
Amazing job :tada: :tada: :tada:  you have met all the requirements for this project
Now you are ready to continue with your next challenge :muscle: 
Don't forget to merge with the master branch

Keep going like this  :clap: :clap:

Oscar Andrés Russi Porras
(for any question you can contact me in slack as @oscarporras02)

**Pull request merged**